### ISSUE TITLE
Use only HTTP status codes

### ISSUE THREAD

**Issue by Friendly-Banana** [Looskie/capybara-api#16]
Hey,
thanks for this wonderful cAPI, I absolutely love it :heart_eyes:.
One small question: Why do you send both HTTP status codes like 200, 500 and also wrap the data into a boolean success?
I find it more straightforward to simply return the data without such a wrapper and treat
every status between [200 -299 as a success](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses), everything else an error.

**Comment by Looskie** [comment]
> Hey, thanks for this wonderful cAPI, I absolutely love it 😍. One small question: Why do you send both HTTP status codes like 200, 500 and also wrap the data into a boolean success? I find it more straightforward to simply return the data without such a wrapper and treat every status between [200 -299 as a success](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses), everything else an error.

Thanks for the kind words! It's just a thing that I do. Using either the `success` or the HTTP response code will work just fine :). It also provides a bit of organization.

Take for example this:
Without wrapping the res in a `data` object:
```ts
{
  message: "test",
  other_key: "test"
}
```

Compared to:
```ts
{
  data: {
    message: "test",
    other_key: "test"
  }
}
```

one reason you would want to wrap it is for conflicting keys when an error pops up, and say that error looks like this
```ts
{
  message: "uh oh",
}
```

Where as if we organize it a bit:

```ts
{
  error: {
    message: "test",
  }
}
```

the problem with the first one, you don't REALLY know if its an error until you actually look at the status codes (which also makes the `success` boolean a nice touch!). Another issue is making strict type interfaces for those cases. It wouldn't be impossible, but would def be harder compared to just organizing it nicely.

TLDR: its just nice for organization purposes to not have data floating in the root object. + the `success` key is nice touch and developer habit that I've grown into.

Let me know if theres anything else! Otherwise I'll close this issue :)
